### DISCUSSION TITLE
Sychronizing from an iOS Qfield project

### DISCUSSION THREAD

**Discussion by joewiseman1** [opengisch/QField#5220] [Q&A]
I am having trouble successfully synchronizing from QField projects in which data has been collected on iOS.

I have managed to package and upload the project into QField onto the iOS device fine and have collected the data successfully. Unlike on Android, on iOS there is no 'export project folder' option to create an export folder. Instead I have created a zipped copy of the project folder within the files app on my iPhone, and copied this back to my desktop where I have unzipped it.

In my master project on the desktop, I then select synchronize from QField in the QFieldSync plugin, and select the unzipped project folder I have copied from my phone. Instead of syncing successfully, I get an error saying that 'the project you imported does not seem to be an offline project'. In doing this it has opened the '_qfield.qgs' version of the project, not the master project to which no data has been synced.

The layers into which data has been added in the field are set to 'offline editing'.

Is there something I'm missing to get it to synchronize back to the master project as I would like?

**Comment by BTreeTech** [comment]
We use a similar workflow (ios and "cable" syncing) but in the QGIS desktop project we change the data source of the layers to point to the contents of the unzipped field package -  its not ideal but our visits are single non repeats and the desktop project is different to the field one.
